roles/setup/tasks/create_permission.yml

```diff
@@ -16,7 +16,8 @@
     method: GET
     status_code:
       - 200
-    url: "{{ auth_keycloak_url }}/admin/realms/{{ realm_name }}/clients/{{ catalog_client['id'] }}/authz/resource-server/scope?deep=false&first=0&max=20&name={{ permission.scope_name }}"
+    url: "{{ auth_keycloak_url }}/admin/realms/{{ realm_name }}/clients/{{ catalog_client['id'] }}\
```

Comment:
I suggest using the more readable way for multiline strings in YAML:
A:
```
url: >-
  {{ auth_keycloak_url }}/admin/realms/{{ realm_name }}/clients/{{ catalog_client['id'] }}
  /authz/resource-server/scope?deep=false&first=0&max=20&name={{ permission.scope_name }}
```
B:
```
url: |
  {{ auth_keycloak_url }}/admin/realms/{{ realm_name }}/clients/{{ catalog_client['id'] }} \
  /authz/resource-server/scope?deep=false&first=0&max=20&name={{ permission.scope_name }}
```

Reply:
@Alex-Izquierdo The only that worked for me is the double quoted strings. The | or > insert either new line or spaces in the string causing a URL parsing error
```
An unknown error occurred: URL can't contain control characters. '/auth/admin/realms/aap/clients/d8b127a3-31f6-44c8-a7e4-4ab9a3e78d95\\n/authz/resource-server/permission/resource\"' (found at least '\\n')",
```

If we find a better way of implementing it we can do that in a different PR